static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -48,7 +53,10 @@ function DomMutations({replay}: Props) {
     setSearchTerm,
   } = useDomFilters({actions});
 
-  const startTimestampMs = replay.getReplay().startedAt.getTime();
+  const currentDomMutation = getPrevReplayEvent({
+    items: items.map(mutation => mutation.crumb),
+    targetTimestampMs: startTimestampMs + currentTime,
+  });
```

Comment:
just comparing this to the console table, when `getPrevReplayEvent` is called.. i can see in the console table we're setting `allowExact: true, allowEqual: true,` and here it's not set.